frontend/testing/playwright/helpers/RouterRoute.ts

```diff
@@ -1,4 +1,5 @@
-﻿import type { Environment } from './StudioEnvironment';
+﻿import { RoutePaths } from 'app-development/enums/RoutePaths';
```

Comment:
This import should not be there as we should avoid having imports from app-development outside app-development 😄

Reply:
Should we move them to shared then? It seems logic to have this enum here at least to ensure that the tests and the code use the same routes 🤔